.adr-dir

```diff
@@ -0,0 +1 @@
+docs/adr
```

Comment:
I've not seen this `.adr-dir` file convention before. What's it for? Should it be included in this PR?

Reply:
i'm using adr tools https://github.com/npryce/adr-tools which auto generates the templates for the ADRs and it uses this file to figure out which folder the ADRs live in

Reply:
will append to the commit message